views/js/controller/DeliveryMgmt/editDelivery.js

```diff
@@ -20,13 +20,56 @@
 define([
     'jquery',
     'util/url',
+    'services/features',
     'ui/modal',
     'css!taoDeliveryRdfCss/delivery-rdf.css'
-], function ($, urlUtil) {
+], function ($, urlUtil, features) {
     'use strict';
 
+    /**
+     * Finds property block and hides it adding .hidden class
+     * @param {string} propName 
+     */
+    function hidePropertyBlockByName(propName) {
```

Comment:
@Nevermind23 @shaveko we are also missing here the `http://www.tao.lu/Ontologies/TAODelivery.rdf#DeliveryTestRunnerFeatures`

Reply:
This is part of this very extension https://github.com/oat-sa/extension-tao-delivery-rdf/blob/c225cee92f19792d41e158a1c35897777069cab3/install/ontology/taodelivery.rdf#L74